lib/l10n/app_id.arb

```diff
@@ -150,14 +150,14 @@
 
   "albumTierNew": "Baru",
   "albumTierPinned": "Disemat",
-  "albumTierSpecial": "Umum",
+  "albumTierSpecial": "Biasa",
   "albumTierApps": "Aplikasi",
   "albumTierRegular": "Lainnya",
 
-  "storageVolumeDescriptionFallbackPrimary": "Penyimpanan internal",
-  "storageVolumeDescriptionFallbackNonPrimary": "kartu SD",
-  "rootDirectoryDescription": "direktori root",
-  "otherDirectoryDescription": "direktori “{name}”",
+  "storageVolumeDescriptionFallbackPrimary": "Penyimpanan Internal",
+  "storageVolumeDescriptionFallbackNonPrimary": "Kartu SD",
+  "rootDirectoryDescription": "Direktori root",
+  "otherDirectoryDescription": "Direktori “{name}”",
```

Comment:
This one (and the one above "rootDirectoryDescription") are used within the sentence "storageAccessDialogMessage", so i don't think adding an uppercase is appropriate. Are you sure of this change?

Reply:
i see those kind of things almost everytime on apps,but if you dont want it i'll remove it